sources/packages/backend/apps/db-migrations/src/sql/DisbursementSchedules/Add-has-estimated-awards-col.sql

```diff
@@ -0,0 +1,35 @@
+-- Set the default to false initially
+ALTER TABLE
+    sims.disbursement_schedules
+ADD
+    COLUMN has_estimated_awards BOOLEAN NOT NULL DEFAULT FALSE;
+
+COMMENT ON COLUMN sims.disbursement_schedules.has_estimated_awards IS 'Indication for whether the disbursement has estimated awards greater than $0.';
+
+-- Update query to calculate TRUE | FALSE based on the data.
+UPDATE
+    sims.disbursement_schedules
+SET
+    has_estimated_awards = CASE
+        WHEN disbursement_total_amounts.value_amount = 0 THEN false
+        ELSE true
+    END
+FROM
+    (
+        SELECT
+            disbursement_schedules.id AS disbursement_schedules_id,

```

Comment:
Minor. for alias please follow singular convention `disbursement_schedules_id` -> disbursement_schedule_id